pkg/smartcontract/manifest/standard/comply.go

```diff
@@ -25,12 +26,21 @@ var checks = map[string][]*Standard{
 // Check checks if manifest complies with all provided standards.
 // Currently only NEP-17 is supported.
 func Check(m *manifest.Manifest, standards ...string) error {
+	return check(m, true, standards...)
+}
+
+// CheckABI is similar to Check but doesn't check parameter names.
+func CheckABI(m *manifest.Manifest, standards ...string) error {
+	return check(m, false, standards...)
+}
+
+func check(m *manifest.Manifest, abiOnly bool, standards ...string) error {
```

Comment:
`abiOnly` seems a bit not-intuitive to me. Maybe something like `checkParamNames` or `considerParamNames`? Because ABI contains parameters names.

Reply:
NeoVM and interops (Contract.Call) doesn't use parameter names, thus they are not a part of ABI. Maybe renaming them will help readability though.